platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema.xml

```diff
@@ -392,290 +331,254 @@
+        <filter class="solr.LowerCaseFilterFactory"/>
+        <filter class="solr.KeywordMarkerFilterFactory" protected="protwords.txt"/>
+        <filter class="solr.EnglishMinimalStemFilterFactory"/>
+        <!-- this filter can remove any duplicate tokens that appear at the same position - sometimes
+             possible with WordDelimiterGraphFilter in conjuncton with stemming. -->
+        <filter class="solr.RemoveDuplicatesTokenFilterFactory"/>
+      </analyzer>
     </fieldType>
 
-    <fieldType name="phonetic" stored="false" indexed="true" class="solr.TextField" >
-        <analyzer>
-            <tokenizer class="solr.StandardTokenizerFactory"/>
-            <filter class="solr.DoubleMetaphoneFilterFactory" inject="false"/>
-        </analyzer>
+    <!-- Just like text_general except it reverses the characters of
+	       each token, to enable more efficient leading wildcard queries.
+    -->
+    <dynamicField name="*_txt_rev" type="text_general_rev"  indexed="true"  stored="true"/>
+    <fieldType name="text_general_rev" class="solr.TextField" positionIncrementGap="100">
+      <analyzer type="index">
+        <tokenizer class="solr.StandardTokenizerFactory"/>
+        <filter class="solr.StopFilterFactory" ignoreCase="true" words="stopwords.txt" />
+        <filter class="solr.LowerCaseFilterFactory"/>
+        <filter class="solr.ReversedWildcardFilterFactory" withOriginal="true"
+                maxPosAsterisk="3" maxPosQuestion="2" maxFractionAsterisk="0.33"/>
+      </analyzer>
+      <analyzer type="query">
+        <tokenizer class="solr.StandardTokenizerFactory"/>
+        <filter class="solr.SynonymGraphFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="true"/>
+        <filter class="solr.StopFilterFactory" ignoreCase="true" words="stopwords.txt" />
+        <filter class="solr.LowerCaseFilterFactory"/>
+      </analyzer>
     </fieldType>
 
-    <fieldType name="payloads" stored="false" indexed="true" class="solr.TextField" >
-        <analyzer>
-            <tokenizer class="solr.WhitespaceTokenizerFactory"/>
-            <!--
-            The DelimitedPayloadTokenFilter can put payloads on tokens... for example,
-            a token of "foo|1.4"  would be indexed as "foo" with a payload of 1.4f
-            Attributes of the DelimitedPayloadTokenFilterFactory :
-             "delimiter" - a one character delimiter. Default is | (pipe)
-       "encoder" - how to encode the following value into a playload
-          float -> org.apache.lucene.analysis.payloads.FloatEncoder,
-          integer -> o.a.l.a.p.IntegerEncoder
-          identity -> o.a.l.a.p.IdentityEncoder
-                Fully Qualified class name implementing PayloadEncoder, Encoder must have a no arg constructor.
-             -->
-            <filter class="solr.DelimitedPayloadTokenFilterFactory" encoder="float"/>
-        </analyzer>
+    <dynamicField name="*_phon_en" type="phonetic_en"  indexed="true"  stored="true"/>
```

Comment:
❓ Any reason for this particular field name? out of context `phon` definitely will be a little trickier to discern, though pretty much any shortened version of this other than the complete word might be difficult.